src/main/java/org/cloudbees/literate/jenkins/promotions/PromotionTrigger.java

```diff
@@ -94,7 +98,11 @@ public FormValidation doCheckJobName(@AncestorInPath Item project, @QueryParamet
 
         public AutoCompletionCandidates doAutoCompleteJobName(@QueryParameter String value) {
             AutoCompletionCandidates candidates = new AutoCompletionCandidates();
-            List<AbstractProject> jobs = Jenkins.getInstance().getItems(AbstractProject.class);
+            Jenkins j = Jenkins.getInstance();
+            if (j == null) {
+                throw new IllegalStateException(); // TODO 1.590+ getActiveInstance
```

Comment:
please add a message - if a user ever sees this having ("Jenkins is starting up or shutting down") is highly valuable.